src/transactions/TransactionTable.js

```diff
@@ -26,30 +27,132 @@ import TransactionForm from './TransactionForm';
  *    filters={filters}/>
  */
 function TransactionTable({ isCreating = false, onStopCreating, filters = {} }) {
-  const transactionList = useSelector((state) => state.transactions.entities.filter(
-    (t) => (t.categoryId === filters.categoryId
-    || filters.categoryId === null
-    || filters.categoryId === undefined)
-    && (t.payee === filters.payee
-    || filters.payee === null
-    || filters.payee === undefined)
-    && (t.date >= filters.startDate
-    || filters.startDate === null
-    || filters.startDate === undefined)
-    && (t.date <= filters.endDate
-    || filters.endDate === null
-    || filters.endDate === undefined),
-  ));
+  const [direction, setdirection] = useState(-1);
+  const [sortProperty, setSortProperty] = useState();
```

Comment:
When you say give it the initial value of 'date' which line(s) do you mean? I have startDate and endDate because I thought it needed that for the computer to understand but also for future programmers that could read this a year later?

Reply:
Ah, sorry, I meant the sortProperty piece of state. Right now you're not giving it any initial value.